pkg/services/forwarder/ports.go

```diff
@@ -387,7 +387,7 @@ func tcpipAddress(nicID tcpip.NICID, remote string) (address tcpip.FullAddress,
 	address = tcpip.FullAddress{
 		NIC:  nicID,
 		Addr: tcpip.AddrFrom4Slice(net.ParseIP(split[0]).To4()),
-		Port: uint16(port),
+		Port: uint16(port), //nolint:gosec
```

Comment:
For this one, using https://pkg.go.dev/strconv#ParseUint instead of `strconv.Atoi` helps to avoid the problem.